components/brave_new_tab_ui/components/block.tsx

```diff
@@ -111,51 +120,58 @@ class Block extends React.Component<Props, {}> {
       style,
       favicon
     } = this.props
-    const starIcon = isBookmarked ? 'fa-star' : 'fa-star-o'
+    const starIcon = isBookmarked ? 'fa-bookmark' : 'fa-bookmark-o'
```

Comment:
@rossmoody worth replacing these font-awesome icons with our own svgs in this PR? I believe we're including the whole font-awesome icon font in the new tab page literally just for these couple glyphs...

Reply:
i was thinking the same thing after investigating it but the application of making it happen seemed a little out of my league to code dependably. i can get the icons together if i could solicit a little assistance. i suppose we could also break that out onto it's own issue if it seems like it could be a tough task too.

Reply:
was able to swap these out after all. i'm unsure if anything else is leveraging the font awesome package but if not I guess it could be omitted now?